packages/website/plugins/add-rule-attributes-list.ts

```diff
@@ -13,191 +13,25 @@ const addRuleAttributesList: Plugin = () => {
     if (rule == null) {
       return;
     }
-    const config = ((): 'recommended' | 'strict' | null => {
-      switch (rule.meta.docs?.recommended) {
-        case 'error':
-        case 'warn':
-          return 'recommended';
-
-        case 'strict':
-          return 'strict';
-
-        default:
-          return null;
-      }
-    })();
-    const autoFixable = rule.meta.fixable != null;
-    const suggestionFixable = rule.meta.hasSuggestions === true;
-    const requiresTypeInfo = rule.meta.docs?.requiresTypeChecking === true;
 
     const parent = root as mdast.Parent;
-    /*
-    This just outputs a list with a heading like:
-
-## Attributes
-
-- [ ] Config
-  - [ ] ✅ Recommended
-  - [ ] 🔒 Strict
-- [ ] Fixable
-  - [ ] 🔧 Automated Fixer (`--fix`)
-  - [ ] 🛠 Suggestion Fixer
-- [ ] 💭 Requires type information
-    */
-    const heading = Heading({
-      depth: 2,
-      text: 'Attributes',
-    });
-    const ruleAttributes = List({
-      children: [
-        NestedList({
-          checked: config != null,
-          children: [
-            ListItem({
-              checked: config === 'recommended',
-              text: '✅ Recommended',
-            }),
-            ListItem({
-              checked: config === 'strict' || config === 'recommended',
-              text: '🔒 Strict',
-            }),
-          ],
-          text: 'Included in configs',
-        }),
-        NestedList({
-          checked: autoFixable || suggestionFixable,
-          children: [
-            ListItem({
-              checked: autoFixable,
-              text: '🔧 Automated Fixer',
-            }),
-            ListItem({
-              checked: suggestionFixable,
-              text: '🛠 Suggestion Fixer',
-            }),
-          ],
-          text: 'Fixable',
-        }),
-        ListItem({
-          checked: requiresTypeInfo,
-          text: '💭 Requires type information',
-        }),
-      ],
-    });
-
     const h2Idx = parent.children.findIndex(
       child => child.type === 'heading' && child.depth === 2,
     );
+    // The actual content will be injected on client side.
+    const attrNode = {
+      type: 'jsx',
+      value: `<rule-attributes name="${file.stem}" />`,
+    } as unknown as mdast.Content;
```

Comment:
This is because this tree is not strictly an MDAST but an MDXAST. It can be fixed by asserting `root`'s type to `unist.Parent` instead. However, in this case we lose stricter types elsewhere. I'll push a version.